### DISCUSSION TITLE
macOS Podman Deskop feedback

### DISCUSSION THREAD

**Discussion by ericcurtin** [containers/podman#22696] [General]
### Bug description

Collected this feedback online:

I have primarily had three issues with Podman, which is really issues with the engine and not the desktop. I am using macOS by the way.

1. When the macOS is idle, and the screen is locked, and I get back to it, any running container is dead, and I have to restart the podman engine.
2. I usually have a tooling container running all day long, and after a few hours I experince the clock inside the container lagging behind the real world.
3. Some i/o intensive operations like running "terraform init", which downloads and unpacks Terraform providers are more than 10 times slower than with Docker Desktop.

Despite these caveats I am happily running Podman and Podman Desktop on my personal mac, but due to these issues I need to run Docker Desktop on my company mac.

Thanks in advance. Podman is a great product, and I would love to see it becoming even better 😀

![image](https://github.com/containers/podman-desktop/assets/1694275/1ebbe863-b9d0-47d3-b3f0-7d364a2fe4ad)

### Operating system

macOS

### Installation Method

None

### Version

1.10.2

**Comment by ericcurtin** [comment]
Not sure if this is the right repo to log it, wasn't sure

**Comment by benoitf** [comment]
moving to Podman repository

**Comment by rhatdan** [comment]
Are we seeing the same issues with podman 5.0 or have some of these been fixed?
